sdk/eventhub/azure-eventhub/azure/eventhub/exceptions.py

```diff
@@ -69,7 +70,10 @@ def __init__(self, message, details=None):
                     self.message += "\n{}".format(detail)
             except:  # pylint: disable=bare-except
                 self.message += "\n{}".format(details)
-        super(EventHubError, self).__init__(self.message)
+        if details and isinstance(details, Exception):
```

Comment:
what happens if  details is None? wondering if we really need this check

Reply:
for clarification, are you saying that we should remove `details` check:
```python
if isinstance(details, Exception):
```
then I agree that `isinstance(details, Exception)` covers `None` check.


or the check `if details and isinstance(details, Exception):` should be completely removed?

Reply:
I have made the check a bit more compact, let me know how it looks to you